mailu/README.md

```diff
@@ -86,6 +86,23 @@ persistence:
 secretKey: chang3m3!
 ```
 
+## Avoiding use of Read-Write Multiple (`RWX`) Volumes
+The traditional configuration of Mailu deployment to Kubernetes is to have a separate pod for each container. This works, but the
+`admin` and `rspamd` pods must write to a shared volume. In turn, the shared volume they write to must contain privileges of
+`RWX` (ReadWriteMany) instead of `RWO` to do so.
```

Comment:
I think this is a common missunderstanding of the Access modes. ReadWriteOnce / ReadWriteMany does not say how many pods can access one volume. It says on how many nodes one volume can be mounted to. So if you set it to ReadWriteOnce, still multiple pods can write to this volume, as long as they are on the same pod. You can use podAffinity to make sure two pods get scheduled on the same node.

Reply:
So it seems like pod affinity rules should be conditionally added when this flag is enabled.

Reply:
Yes, I would suggest keeping the two pods - admin and rspamd - split up like they were before and just add a default podAffinity in the valuey.yaml that schedules them on the same node. If someone wants to change this he could just override the podAffinity then.

Some thoughts:
I would suggest avoiding hard to think about features like `avoidRWXVolumes`, instead just make the storage of every pod customizable, with automatic PVC creation per pod, and the ability to specify a custom pre-existing PVC for each pod.
I think the less magic the better because every one has different needs.

Reply:
Agreed. It also provides a better upgrade path if someone starts with the pod affinity rules and later wants to remove them in a cutover to RWO because of scale issues.

After this discussion, I think you have a better plan if you still want to put together your PR. We can do the same process on your branch and get it pushed.

Reply:
Well, since you already have a PR open now, I could just rebase my proposed changes into your branch and then you can review it. Just give me a few days to do it. You can make a tag or a new branch as backup :)

Thanks!

Reply:
or I open a new PR, however you like...

Reply:
It's mostly out of respect for the originality of your work.

Reply:
And you already did a lot of work, I don't want to destroy that.

Reply:
Haha, we are in what we call "heated agreement" now! 😂😂

I didn't expect what I did was necessary at this point, but if you think some of it is, go ahead and apply your changes here. Makes good sense that way to me.